docker-compose.yml

```diff
@@ -206,3 +206,10 @@ services:
         ports:
             - "${TEST_MONGODB_STANDALONE_PORT:-27017}:27017"
         command: mongod --port 27017  --oplogSize 16
+
+    percona-mongo-standalone:
```

Comment:
This container is not used for any test. You need to integrate it with existing tests to make this addition useful.

Reply:
Ok, got it

Removed